tests/std/tests/P0811R3_midpoint_lerp/test.cpp

```diff
@@ -1031,6 +1031,8 @@ bool test_lerp() {
     return true;
 }
 
+#pragma warning(push)
+#pragma warning(disable : 4756) // ignore constant arithmetic overflow warning

```

Comment:
if the optimizer is triggering this warning, doesn't that mean this is a faulty warning, or a faulty optimizer?

Reply:
Ah, the age old trick of looking further down the diff. Thanks, this checks out.